### ISSUE TITLE
metric depth estimation vs metric3d

### ISSUE THREAD

**Issue by MaybeShewill-CV** [LiheYoung/Depth-Anything#42]
Does the method have detailed results on metric measurements, especially in comparison with the metric3d method?

**Comment by LiheYoung** [comment]
Hi, thank you for your question.

We primarily focus on *relative* depth estimation. So we mainly compare Depth Anything with the latest and previously strongest MiDaS v3.1 BEiT model. And when transferring our relative MDE model to *metric* MDE, we follow ZoeDepth to directly fine-tuning with metric depth labels, without using the camera intrinsics as Metric3D does. So we do not compare with Metric3D, but compare with ZoeDepth for fairness.

Last but not least, our work is more of a relative depth estimation work, following the MiDaS research line. We believe the users can fine-tune our well-performed pre-trained model with their own metric depth data for better metric depth estimation.

**Comment by MaybeShewill-CV** [comment]
@LiheYoung Thanks for reply:)
